client/src/fabric/tensegrity-fabric.ts

```diff
@@ -37,6 +34,12 @@ export enum Selectable {
     GROW_FACE,
 }
 
+interface IFabricOutput {
+    name: string
+    joints: Array<{ index: string, x: string, y: string, z: string }>
```

Comment:
does the linter not complain that you should use T[] instead of Array<T> here?

Reply:
On the contrary! The type system doesn't seem to allow the shorthand here. I had to switch.